source/index.ts

```diff
@@ -135,10 +135,13 @@ namespace is { // tslint:disable-line:no-namespace
 	export const symbol = isOfType<Symbol>('symbol');
 	// tslint:enable:variable-name
 
+	export const nullOrUndefined = (value: any): value is null | undefined => null_(value) || undefined(value);
+	export const numericString = (value: any): value is String =>
```

Comment:
if this is supposed to be a `numericString` how come you accept `any`? and how does it indicate that `value is String`?
First, use `string` over `String`, second - I believe this is not the correct signature for this method.
I think that it should just return bool, since you can't really create a type guard that assures you that `string` is typed as a `number`, you can just have a bool that tells you that, and you can just `Number(_)` it...
```suggestion
	export const numericString = (value: string | number): bool =>
```

the `|  number` is optional, only if you feel that's necessary...